cmd/ctr/commands/run/run_windows.go

```diff
@@ -35,6 +35,10 @@ var platformRunFlags = []cli.Flag{
 		Name:  "isolated",
 		Usage: "run the container with vm isolation",
 	},
+	cli.BoolFlag{
```

Comment:
Since these are platform specific flags, if all environments now have this flag it should be added in `run.go` and removed from `run_unix.go`

Reply:
thanks, changed